scripts/scripts.js

```diff
@@ -58,7 +78,16 @@ export function decorateMain(main) {
  * loads everything needed to get to LCP.
  */
 async function loadEager(doc) {
-  document.documentElement.lang = 'en';
+  const preferredLanguage = navigator.languages.find(
+    (l) => SUPPORTED_LANGUAGES.includes(l),
+  ) || DEFAULT_LANGUAGE;
+  if (window.location.pathname === '/' && window.location.origin.match(/\.hlx\.(page|live)$/)) {
+    window.location.replace(`/${preferredLanguage}/`);
```

Comment:
does this mean `/` will redirect to `/en` or `/de` (depending on the default language)? Currently customer website seems to have `de` as default without any lang code, and rest with lang codes:

- `https://aerotop-sg.com/technische-daten/`
- `https://aerotop-sg.com/en/technical-data/`